faction.cpp

```diff
@@ -36,13 +37,13 @@ char const *as[] = {
 
 char const **AttitudeStrs = as;
 
-char const *fs[] = {
-	"War",
-	"Trade",
-	"Magic"
-};
+const std::string F_WAR = "War";
```

Comment:
why do we changing this to std string?

Reply:
So that there is no unexpected behavior in using strings with STD containers. `std::string` is better in all aspects and there are no problems to convert it into `AString`.

`std::string` is available since 1998. It existed already when Atlantis code was written. `AString` should not exist then, and it is wrong to use it now.